client/env.d.ts

```diff
@@ -4,9 +4,14 @@ declare global {
   namespace NodeJS {
     interface ProcessEnv {
       REACT_APP_ENVIRONMENT: string;
+      REACT_APP_URL: string;
+      REACT_APP_URL_DTLINK: string;
       REACT_APP_URL_LIB: string;
```

Comment:
by the same convention, this should be `REACT_APP_URL_LIBLINK`

Reply:
Oh yeah, think I missed it in a blunder. I'll make a quick update.